### ISSUE TITLE
Many error entries in the server-main.log

### ISSUE THREAD

**Issue by Mandikor** [splitlungs/krpgenchantment#1]
One of many error messages when attacking with enchanted Paxel on different entities.

```
21.11.2024 14:47:41 [Error] Exception: Method not found: 'Vintagestory.API.Common.IInventory Vintagestory.API.Common.EntityAgent.get_GearInventory()'.
   at KRPGLib.Enchantment.EnchantmentEntityBehavior.DamageEntity(Entity byEntity, ItemStack stack, EnumEnchantments enchant, Int32 power)
   at KRPGLib.Enchantment.EnchantmentEntityBehavior.TryEnchantment(EntityAgent byEntity, ItemStack stack, String enchant, Int32 power) in D:\VSProjects\Net7\KRPGEnchantment\krpgenchantment\Behaviors\EnchantmentEntityBehavior.cs:line 261
   at KRPGLib.Enchantment.EnchantmentEntityBehavior.TryEnchantments(EntityAgent byEntity, ItemStack stack, Dictionary`2 enchants) in D:\VSProjects\Net7\KRPGEnchantment\krpgenchantment\Behaviors\EnchantmentEntityBehavior.cs:line 250
   at KRPGLib.Enchantment.EnchantmentEntityBehavior.OnInteract(EntityAgent byEntity, ItemSlot itemslot, Vec3d hitPosition, EnumInteractMode mode, EnumHandling& handled) in D:\VSProjects\Net7\KRPGEnchantment\krpgenchantment\Behaviors\EnchantmentEntityBehavior.cs:line 233
   at Vintagestory.API.Common.EntityAgent.OnInteract(EntityAgent byEntity, ItemSlot slot, Vec3d hitPosition, EnumInteractMode mode) in VintagestoryApi\Common\Entity\EntityAgent.cs:line 288
   at Vintagestory.Server.ServerSystemEntitySimulation.HandleEntityInteraction(Packet_Client packet, ConnectedClient client)
   at Vintagestory.Server.ServerMain.ProcessNetMessage(NetIncomingMessage msg, NetServer mainSocket) in VintagestoryLib\Server\ServerMainNetworking.cs:line 141

```

**Comment by splitlungs** [comment]
Can you provide Game version, KRPG Enchantment version, and Paxel version?

**Comment by Mandikor** [comment]
Since I'm currently switching to the new v1.20, I tested my Paxel Mod with the last v1.20-pre.13. The KRPG Enchantment Mod ran in version v5.0 and then with v5.1
I just wanted to see whether all the mods that I support with my Paxel Mod also run with VS 1.20. The 2 typographical errors are relatively harmless and quickly resolved.
The reported bug above had no visible impact in the game. It wasn't until I was browsing through the log files that I noticed them. But I was able to narrow it down enough to trigger them when I hit creatures with enchanted tools.
